manifests/p/PatrickMortara/WIA-Loader/2.6.4.0/PatrickMortara.WIA-Loader.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PatrickMortara.WIA-Loader
+PackageVersion: 2.6.4.0

```

```suggestion
PackageVersion: WIA-Loader 2.6.4.0
```
This must match Control Panel.